update several document related to privilege sql statement (#2828)
pingcap/docs: sql-statements/sql-statement-revoke-privileges.md

# `REVOKE <privileges>`

This statement removes privileges from an existing user.

> Executing this statement require `GRANT OPTION` privilege and all peivileges you want to revok.

Reviewer: ```suggestion
Executing this statement requires the `GRANT OPTION` privilege and all privileges you revoke.
```
